zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -85,48 +88,42 @@ class MeasurementType(enum.IntFlag):
             attr=ElectricalMeasurement.AttributeDefs.rms_current_ph_c.name,
             config=REPORT_CONFIG_OP,
         ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_b.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_c.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
         AttrReportConfig(
             attr=ElectricalMeasurement.AttributeDefs.rms_voltage.name,
             config=REPORT_CONFIG_OP,
         ),
         AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_voltage_max.name,
-            config=REPORT_CONFIG_DEFAULT,
+            attr=ElectricalMeasurement.AttributeDefs.rms_voltage_ph_b.name,
+            config=REPORT_CONFIG_OP,
         ),
         AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.ac_frequency.name,
+            attr=ElectricalMeasurement.AttributeDefs.rms_voltage_ph_c.name,
             config=REPORT_CONFIG_OP,
         ),
         AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.ac_frequency_max.name,
-            config=REPORT_CONFIG_DEFAULT,
+            attr=ElectricalMeasurement.AttributeDefs.ac_frequency.name,
+            config=REPORT_CONFIG_OP,
         ),
     )
     ZCL_INIT_ATTRS = {
         ElectricalMeasurement.AttributeDefs.ac_current_divisor.name: True,
         ElectricalMeasurement.AttributeDefs.ac_current_multiplier.name: True,
+        ElectricalMeasurement.AttributeDefs.ac_frequency_divisor.name: True,
+        ElectricalMeasurement.AttributeDefs.ac_frequency_max.name: False,
+        ElectricalMeasurement.AttributeDefs.ac_frequency_multiplier.name: True,
         ElectricalMeasurement.AttributeDefs.ac_power_divisor.name: True,
         ElectricalMeasurement.AttributeDefs.ac_power_multiplier.name: True,
         ElectricalMeasurement.AttributeDefs.ac_voltage_divisor.name: True,
         ElectricalMeasurement.AttributeDefs.ac_voltage_multiplier.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_frequency_divisor.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_frequency_multiplier.name: True,
+        ElectricalMeasurement.AttributeDefs.active_power_max.name: False,
         ElectricalMeasurement.AttributeDefs.measurement_type.name: True,
         ElectricalMeasurement.AttributeDefs.power_divisor.name: True,
+        ElectricalMeasurement.AttributeDefs.power_factor.name: False,
+        ElectricalMeasurement.AttributeDefs.power_factor_ph_b.name: False,
+        ElectricalMeasurement.AttributeDefs.power_factor_ph_c.name: False,
```

Comment:
Per [this comment](https://github.com/zigpy/zha/pull/339#discussion_r1917450016), we should be able to make use `True` for the cache here.
They are [polled using themEM cluster at runtime](https://github.com/zigpy/zha/blob/a833f43c0c3507c62b4331ae102e924e86bbb96c/zha/zigbee/cluster_handlers/homeautomation.py#L132-L149), which checks for `unsupported_attributes`, so we can avoid the extra polling during during each startup, which doesn't check for `unsupported_attributes` (yet).
Even if we eventually add it, we can avoid polling this at startup, since we'll poll it after a few seconds/minute anyway.